media/system/js/frontediting.js

```diff
@@ -7,7 +7,7 @@
  * JavaScript behavior to add front-end hover edit icons with tooltips for modules and menu items.
  *
  */
-(function($) {
+jQuery(function ($) {
```

Comment:
When you run (function($) { ... it executes immediately requiring to have document ready inside and when you are using 1) jQuery(function ...  it wait for DOM ready. Before and after the modification, it still uses jQuery so I don't see any benefit in refactoring. If you can use the before editing version, having a native handler for document ready (Check this Stackoverflow question https://stackoverflow.com/questions/799981/document-ready-equivalent-without-jquery) you can replace jQuery.